src/pages/Contact.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import '../style/main.scss'
```

Comment:
I don't think you need to include `main.scss` on this page (or any other pages). It's enough so you include it in App.js.